draft-ietf-ppm-dap.md

```diff
@@ -687,8 +761,6 @@ cipher suites (a la OHTTP/ECH).]
 The client MUST abort if any of the following happen for any HPKE config
 request:
 
-* the client and aggregator failed to establish a secure,
-  aggregator-authenticated channel;
```

Comment:
I deleted this because it's implicit by virtue of using HTTPS: you won't even be able to establish a connection if you can't verify server identity, so we don't need more protocol/application layer language to repeat features offered by the transport.